components/tabBarComponents/EventTabBar.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import { Ionicons } from "@expo/vector-icons"
+import { BlurView } from "expo-blur"
+
+export const EventTabBar = () => {
+  const darkIconColor = "#1c1c1c"
+  const iconSize = 24
+
+  return (
+    <BlurView intensity={100} tint="light" style={styles.blur}>
+      <View style={styles.container}>
+        <TouchableOpacity style={styles.icon}>
```

Comment:
We'll need some hitSlop for each button to make them easier to tap. Cover in a follow up PR